Package.resolved

```diff
@@ -0,0 +1,43 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "CwlCatchException",
+        "repositoryURL": "https://github.com/mattgallagher/CwlCatchException.git",
```

Comment:
Where did this come from?

Reply:
I think this is part of [Nimble](https://github.com/Quick/Nimble/blob/master/Package.swift) -> [CwlPreconditionTesting](https://github.com/mattgallagher/CwlPreconditionTesting/blob/master/Package.swift)